flink-core/src/main/java/org/apache/flink/metrics/MetricRegistry.java

```diff
@@ -198,8 +220,13 @@ public void register(Metric metric, String metricName, AbstractMetricGroup group
 	 */
 	public void unregister(Metric metric, String metricName, AbstractMetricGroup group) {
 		try {
-			if (reporter != null) {
-				reporter.notifyOfRemovedMetric(metric, metricName, group);
+			if (reporters != null) {
+				for (MetricReporter reporter : reporters) {
+
```

Comment:
let's remove this empty line so the code is formatting-wise identical to `register()`